### PR TITLE
Some cleanup of voting page

### PR DESCRIPTION
**Pull request by pksjce**
- [x] Cleaned up login part.
- [x] Voting arrows are only one per side now. Maybe @NejcZdovc  could implement #444  on these?

Screenshots
**Logged In**
screen shot 2016-11-28 at 7 29 34 pm

**Logged out**
screen shot 2016-11-28 at 7 31 46 pm

**On Desktop**
screen shot 2016-11-29 at 8 15 43 am
screen shot 2016-11-29 at 8 16 01 am

### CODE REVIEW

**Comment by pksjce** [comment]
Sure will do that.

**Comment by NejcZdovc** [comment]
@pksjce sure I can implement into this design

File: components/vote/app-style.scss [hunk 1]
```diff
@@ -5,7 +5,20 @@
 .vote-app {
   margin: 1.5em;
 
-  &__influence-descriptions {
+  &__top {
+      display: flex;
```
Comment (sokra) [inline components/vote/app-style.scss]: spacing

File: components/vote/button/button.jsx [hunk 2]
```diff
@@ -2,7 +2,8 @@ import React from 'react';
 require("./button-style.scss");
 
 export default (props) => {
-  let { value, myValue, maxDown, maxUp, color, onVote, className } = props;
+  let { value, myValue, maxDown, maxUp, color, onVote, className, isLoginActive } = props;
+  let {voteAppToken} = localStorage;
```
Comment (sokra) [inline components/vote/button/button.jsx]: We should not let this component know about the login system (`localStorage`, `voteAppToken`). Could this information just be passed via props i. e `loggedIn = true` or `myValue != null`?

File: components/vote/button/new-button.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      </a>;
+    } else {
+      return <a
+        className={className}>
+          {fn({size: size, color: "#eee"})}
+      </a>;
+    }
+  }
+  render() {
+    const {voteAppToken} = localStorage;
```
Comment (sokra) [inline components/vote/button/new-button.jsx]: We should not let this component know about the login system (localStorage, voteAppToken). Could this information just be passed via props i. e loggedIn = true or myValue != null?

File: components/vote/app.jsx [hunk 4]
```diff
@@ -5,7 +5,9 @@ import * as api from "./api";
 import './app-style';
 import './influence-style';
 import VoteButton from './button/button';
```
Comment (sokra) [inline components/vote/app.jsx]: Remove this if it's no longer used. Also delete the file if no longer used.

**Comment by sokra** [comment]
Could you include a non-mobile screenshot?

File: components/vote/button/new-button.jsx (hunk 3 above)
Comment (pksjce) [inline components/vote/button/new-button.jsx]: Sure. Will do that

**Comment by sokra** [comment]
We should merge this into a separate branch so @NejcZdovc can do his changes.

**Comment by pksjce** [comment]
I have given @NejcZdovc collaborator permissions to my fork. Does that help?

**Comment by NejcZdovc** [comment]
@pksjce I think this will work. I will start working on `voting/logoutstate` branch.

**Comment by NejcZdovc** [comment]
I added my new vote app logic and also I added some mobile modifications, so that it's easy to read on mobile. One thing that is problematic is that arrow is too small for mobile and it's hard to vote. Any suggestion?

What if we place arrows on the side of the vote number (up arrow on the left and down arrow on the right)?

**Comment by pksjce** [comment]
@NejcZdovc - Your changes look good!

**Comment by bebraw** [comment]
Good to merge now?

**Comment by pksjce** [comment]
I like the behaviour right now, but may have to change the up and down buttons to have better haptic feedback.

**Comment by bebraw** [comment]
Yeah, another PR would work.

**Comment by pksjce** [comment]
Thanks!

**Pull request merged**